rubin_scheduler/scheduler/example/example_scheduler.py

```diff
@@ -1346,8 +1381,8 @@ def main(args):
     neo_night_pattern = pattern_dict[neo_night_pattern]
     reverse_neo_night_pattern = [not val for val in neo_night_pattern]
 
-    # Generate the rolling footprint for this start of survey
-    sky = EuclidOverlapFootprint(nside=nside)
+    # Modify the footprint
+    sky = EuclidOverlapFootprint(nside=nside, smc_radius=4, lmc_radius=6)
```

Comment:
smc_radius and lmc_radius -- are these the defaults for EuclidOverlapfootprint? If not, the footprint maybe needs an update?